src/modal/modal-config.ts

```diff
@@ -75,4 +81,5 @@ export interface NgbModalOptions {
 export class NgbModalConfig implements NgbModalOptions {
   backdrop: boolean | 'static' = true;
   keyboard = true;
+  focusFirst = true;
```

Comment:
My understanding was that we don't want to introduce this option for now. Did anything change?